numcodecs/checksum32.py

```diff
@@ -10,7 +10,7 @@
 from .jenkins import jenkins_lookup3
 
 if TYPE_CHECKING:
```

Comment:
Should we move this to the branch itself?

```suggestion
if TYPE_CHECKING:  # pragma: no cover
```